tests/unit/wmg_processing/test_marker_gene_cube_creation.py

```diff
@@ -0,0 +1,33 @@
+from backend.wmg.pipeline.summary_cubes.marker_genes import create_marker_genes_cube
+import unittest
+from tests.unit.backend.wmg.fixtures import FIXTURES_ROOT
+import os
+import tiledb
+import shutil
+
+TEST_SNAPSHOT = "test-fmg-snapshot"
+
+# creates the marker gene cube de novo and compares to the existing fixture
```

Comment:
Worth commenting also that the the marker computation logic is being tested elsewhere and that this test _only_ needs to verify the snapshot cube building is working as expected.   (Before I realized that, I wrote a comment about how this was not a sufficient test to ensure marker computation logic is not really being tested by this.)